aiida/cmdline/commands/cmd_computer.py

```diff
@@ -539,7 +538,24 @@ def computer_delete(computer):
     echo.echo_success(f"Computer '{label}' deleted.")
 
 
-@verdi_computer.group('configure')
+class LazyConfigureGroup(click.Group):
+    """A click group that will lazily load the subcommands for each transport plugin."""
+
+    def list_commands(self, ctx):
+        subcommands = super().list_commands(ctx)
+        subcommands.extend(get_entry_point_names('aiida.transports'))
```

Comment:
Am I right in assuming that this will result in the list of subcommands being recreated every time `list_commands` is called?

Why not trigger`computer_configure.add_command` the first time either `list_commands` or `get_command` are called?

Reply:
Because I don't want to have to load all the transport plugins, just to list the commands; `add_command` should only be called when you actually need that command.
When you say "every time list_commands is called", why would you expect list_commands to be called multiple times? 
Also note `get_entry_point_names` is cached and so fast on re-calls

Reply:
> Because I don't want to have to load all the transport plugins, just to list the commands; `add_command` should only be called when you actually need that command.

Sure, that part I get.

> When you say "every time list_commands is called", why would you expect list_commands to be called multiple times?

Hm... for example in the test suite?
I don't know whether it does happen in regular use, perhaps not.

Reply:
Yeh I’d say every time you call the CLI obviously you are reloading everything, so there would not really be much possibility of multiple calls

Reply:
Ok, I wouldn't call this implementation ideal but it fixes the speed issue and it works, so I guess it's ok.

Feel free to merge or to further improve